src/chrome/content/rules/Uvnc.com.xml

```diff
@@ -0,0 +1,15 @@
+<!--
+	Nonfunctional hosts in *.uvnc.com:
+
+	h: http redirect
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="Uvnc.com">
+	<target host="uvnc.com" />
```

Comment:
https://travis-ci.org/EFForg/https-everywhere/jobs/586111710#L319
No ^

Reply:
What do I need to do? Where to set this "^"? BTW, it's redirecting to https://www.uvnc.com/, maybe this ruleset isn't needed anymore.

Reply:
^ means `uvnc.com`.
Travis test tell us there is no `uvnc.com`
Remove `<target host="uvnc.com" />` to comment.